src/api/routers/participantsRouter.ts

```diff
@@ -277,6 +277,15 @@ export function createParticipantsRouter() {
     }
   );
 
+  participantsRouter.post(
+    '/:participantId/completedRecommendationsWorkflow',
+    async (req: ParticipantRequest, res: Response) => {
+      const { participant } = req;
+      await participant!.$query().update({ completedRecommendationsWorkflow: true });
+      return res.status(200).json(participant);
```

Comment:
Wasn't sure what to return here, so just returned the whole participant. Happy to change to something simpler: maybe just `participant.completedRecommendationsWorkflow`?

Reply:
Changed to just return `participant.completedRecommendationsWorkflow`

Reply:
Changed again to return the whole participant :)